app/src/app/[locale]/admin/products/create/_components/create-product-form.tsx

```
@@ -107,30 +108,51 @@ export const CreateProductForm = () => {
-
             <div className='grid grid-cols-2 gap-4'>
+              <FormField
+                control={form.control}
+                name='price'
+                render={({ field }) => (
+                  <FormItem>
+                    <FormLabel>Prix</FormLabel>
+                    <FormControl>
+                      <Input
+                        onFocus={(e) =>
+                          e.target.value === '0' && (e.target.value = '')
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)